xgcm/grid.py

```diff
@@ -1424,15 +1452,6 @@ def interp_like(self, array, like, boundary=None, fill_value=None):
         )
         return array
 
-    def _interp_metric(self, da, axes):
```

Comment:
I think this functionality will be implemented once interp_like is combined with get_metric.

Reply:
yes lets keep this in another PR.